### PR TITLE
Corral-packages maintenance for registries + rancher (rke1/rke2/k3s)

### PR DESCRIPTION
**Pull request by Josh-Diamond**
In order to properly test out these changes e2e, I've combined the work from https://github.com/rancherlabs/corral-packages/pull/53 + https://github.com/rancherlabs/corral-packages/pull/54, into a single branch, as both changes are needed to test e2e

### CODE REVIEW

**Review by markusewalker — COMMENTED** [review]
Overall, LGTM. Just waiting for freeform job.

**Review by markusewalker — APPROVED** [review]
Used this branch in issue validation this week. As such, approving.

File: templates/k3s/overlay/opt/corral/k3s/init-cluster.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ config="write-kubeconfig-mode: 644
 cluster-init: true
 tls-san:
   - ${CORRAL_api_host}
+  - ${CORRAL_kube_api_host}
```
Comment (igomez06) [inline templates/k3s/overlay/opt/corral/k3s/init-cluster.sh:10]: Just making sure, why is this being added as well?

**Review by igomez06 — COMMENTED** [review]
Just one question about the init-cluster.sh for both rke2 and k3s

**Review by igomez06 — APPROVED** [review]
LGTM

**Pull request merged**